tests/test.py

```diff
@@ -25,31 +25,31 @@
 # from resurfemg.helper_functions import bad_end_cutter
 # from resurfemg.helper_functions import bad_end_cutter_better
 # from resurfemg.helper_functions import bad_end_cutter_for_samples
-from resurfemg.helper_functions import count_decision_array
+from resurfemg.helper_functions.helper_functions import count_decision_array
```

Comment:
(Almost) all imported functions from resurfemg.helper_functions.helper_functions are not used anymore as the tests using them are commented out and moved to another test file. Also comment them out here instead of changing them. This goes for:
- count_decision_array
- zero_one_for_jumps_base
- working_pipeline_exp
- relative_levenshtein
- scale_arrays
- distance_matrix